signals/commands/things/src/test/java/org/eclipse/ditto/signals/commands/things/modify/DeleteAttributeResponseTest.java

```diff
@@ -36,6 +39,8 @@
  */
 public final class DeleteAttributeResponseTest {
 
+    private static final JsonPointer INVALID_JSON_POINTER = JsonFactory.newPointer("key1/äöü/foo");
```

Comment:
Shouldn't TestConstants.INVALID_JSON_POINTER be used instead? (Same value)

Reply:
Yes!